lib/private/DB/Adapter.php

```diff
@@ -46,11 +46,10 @@ public function fixupStatement($statement) {
 	/**
 	 * Create an exclusive read+write lock on a table
 	 *
-	 * @param string $tableName
 	 * @throws Exception
 	 * @since 9.1.0
 	 */
-	public function lockTable($tableName) {
+	public function lockTable(string $tableName) {
```

Comment:
When adding types that we can also add the return type

```suggestion
	public function lockTable(string $tableName): void {
```

Reply:
That gave me a psalm error... 😞 See initail commit at https://github.com/nextcloud/server/pull/47675/commits/dfcf3132dda2c74ca96a3972ae8d78b4eba55b88

Reply:
Ah yes, because all the implementing classes (AdapterSqlite etc) also would need that typing.